fs/ftpfs.py

```diff
@@ -235,6 +235,13 @@ def read(self, size=-1):
                 remaining -= len(chunk)
         return b"".join(chunks)
 
+    def readinto(self, buffer):
```

Comment:
@willmcgugan Should this be in the base-class, rather than ftpfs?

Reply:
`FTPFile` inherits from `io.RawIOBase` which raises `NotImplementedError` when `readinto` is called.

`fs.iotools.RawWrapper` does have a `readinto` method which provides a default implementation. However it does not catch `NotImplementedError` which is raised when a subclass of `io.RawIOBase` falls back to the `RawIOBase` default method.

It might be good for `fs.iotools.RawWrapper` to catch `NotImplementedError` for `readinto`.

Reply:
But even adding the catch in `RawWrapper` doesn't cover all cases.  `FTPFS.openbin` returns a `FTPFile` (as far as I can tell) so it is still nice to have the `readinto` method on `FTPFile`.

Reply:
Fair enough, you seem to have looked into it in much more detail than me :eyes: :+1: